Src/Web/Web.Nuget/Resources/ApplicationInsights.config.install.xdt

```diff
@@ -43,17 +43,4 @@
     </Add>
     <Add xdt:Transform="InsertIfMissing" xdt:Locator="Match(Type)" Type="Microsoft.ApplicationInsights.Web.ExceptionTrackingTelemetryModule, Microsoft.AI.Web" />
   </TelemetryModules>
-
-  <TelemetryProcessors xdt:Transform="Remove" xdt:Locator="Condition(count(*)=0)"/>
-
-  <TelemetryProcessors xdt:Transform="InsertIfMissing">
-    <Add xdt:Transform="Remove" xdt:Locator="Condition(contains(@Type, Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.AdaptiveSamplingTelemetryProcessor))" />
```

Comment:
We do not need to have Removes in install. Install should only add and uninstall  - remove. During update uninstall of a previous nuget is run.